backend/src/main/java/com/quizzard/quizzard/repository/QuestionAnswerRepository.java

```diff
@@ -10,7 +10,10 @@
 
 @Repository
 public interface QuestionAnswerRepository extends JpaRepository<QuestionAnswer, Long> {
-    List<QuestionAnswer> findAllByQuizAttemptId(Long quizAttemptId);
```

Comment:
I believe that this should work. Have you experienced any issue with this function.

Reply:
Actually, I did not try it. However, when I look at it now, this function and the one I wrote look the same. Probably, it also work 😄